### PR TITLE
Closes #6877 Create a new tab in WP Rocket

### LINKED ISSUE #6877: Create a New Tab in WP Rocket
Description and assets available [here](https://docs.google.com/document/d/1lNfOB2dUg4FtDeW8OInwKP5GIExBNGgHXenqdJ8hFa8/edit#heading=h.wdkofbwe72cn)

### PR DESCRIPTION
**Pull request by Khadreal**
# Description

Fixes #6877
Add a new tab to WP Rocket settings page

## Type of change
- [x] Enhancement (non-breaking change which improves an existing functionality).

## Detailed scenario
There should be a new tab on the settings page

### Documentation
Added a new tab on settings page to view plugin.

# Mandatory Checklist

## Code validation

- [x] I validated all the Acceptance Criteria. If possible, provide screenshots or videos.
- [x] I triggered all changed lines of code at least once without new errors/warnings/notices.

## Code style
- [x] I wrote a self-explanatory code about what it does.
- [x] I protected entry points against unexpected inputs.
- [x] I did not introduce unnecessary complexity.

### CODE REVIEW

**Comment by Khadreal** [comment]
We have different state for the plugins and in this screenshot, we have install, activate and activated plugins.

Screenshot 2024-10-21 at 17 46 48

File: assets/js/lazyload-css.min.js
Comment (jeawhanlee) [inline assets/js/lazyload-css.min.js:1]: Why was this file modified in this PR?

File: assets/js/wpr-beacon.js
Comment (jeawhanlee) [inline assets/js/wpr-beacon.js:1]: Why was this file modified in this PR?

File: assets/js/wpr-beacon.min.js
Comment (jeawhanlee) [inline assets/js/wpr-beacon.min.js:1]: Why was this file modified in this PR?

File: assets/js/wpr-beacon.min.js.map
Comment (jeawhanlee) [inline assets/js/wpr-beacon.min.js.map:1]: Why was this file modified in this PR?

File: inc/Engine/Admin/Settings/Render.php [hunk 1]
```diff
@@ -194,6 +215,34 @@ public function render_tools_section() {
...
+		];
+
+		$reordered_data = [];
+
+		// Re order data.
+		foreach ( $order as $key ) {
+			if ( isset( $data[ $key ] ) ) {
+				$reordered_data[ $key ] = $data[ $key ];
+			}
+		}
```
Comment (jeawhanlee) [inline inc/Engine/Admin/Settings/Render.php]: We don't need to reorder, we already do that from the package, but we also order based on activated status following the AC [here](https://docs.google.com/document/d/1lNfOB2dUg4FtDeW8OInwKP5GIExBNGgHXenqdJ8hFa8/edit#heading=h.5s7nn046slkw)

**Review by jeawhanlee — CHANGES_REQUESTED** [review]

File: inc/Engine/Admin/Settings/Render.php (hunk 1 above)
Comment (Khadreal) [reply inc/Engine/Admin/Settings/Render.php]: I reordered it based on the design

File: assets/js/lazyload-css.min.js
Comment (Khadreal) [reply assets/js/lazyload-css.min.js:1]: ran gulp for js and commits all. will remove them

**Review by jeawhanlee — APPROVED** [review]

**Review by remyperona — APPROVED** [review]

**Review by Miraeld — APPROVED** [review]

File: views/settings/page-sections/plugins.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+				<div class="wpr-optionHeader">
+					<h3 class="wpr-title2"><?php echo esc_html( $category['title'] ); ?></h3>
+				</div>
+				<div class="wpr-Page-row wpr-plugins">
+					<?php foreach ( $category['plugins'] as $index => $wpm_plugin ) : // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound ?>
+					<div class="wpr-Page-col-half wpr-plugins--box">
+						<div class="wpr-plugins--logo">
+							<img src="<?php echo esc_url( WP_ROCKET_ASSETS_IMG_URL . $wpm_plugin['logo']['file'] ); ?>"
+								alt="<?php echo esc_attr( $wpm_plugin['title'] ); ?>"
+								style="width: <?php echo esc_attr( $wpm_plugin['logo']['width'] ); ?>" />
```
Comment (remyperona) [inline views/settings/page-sections/plugins.php]: add `loading="lazy"` attribute to the images

**Review by hanna-meda — APPROVED** [review]
Everything looks and works as it should now. Thank you, @Khadreal.
Test Run Results below:
[testrail-report-653.pdf](https://github.com/user-attachments/files/17546066/testrail-report-653.pdf)

**Pull request merged**